app_feup/lib/model/entities/time_utilities.dart

```diff
@@ -1,7 +1,39 @@
+import 'package:intl/intl.dart';
+
+const weekdays = [
+  'N/A',
+  'Segunda',
+  'Terça',
+  'Quarta',
+  'Quinta',
+  'Sexta',
+  'Sábado',
+  'Domingo'
+];
```

Comment:
Should be a static variable of the `TimeString` class?
Also, please create an issue to ensure other entities make use of this, such as `Lecture`.

Reply:
done!

About the Lecture, there are some differences. Lecture uses "Segunda-feira" not "Segunda". 
Do I change to lecture format? 
It's something to discuss on meeting

Another thing is about the 'N/A'. I am thinking to remove it an change `weekdays[this.weekday]` to  `weekdays[this.weekday-1]`. What do you think?

Reply:
Assuming `DateTime` does always store a valid weekday, I think it makes sense